pkg/model/manifest_test.go

```diff
@@ -1382,3 +1393,194 @@ func TestSecretValidate(t *testing.T) {
 		})
 	}
 }
+
+func TestRead(t *testing.T) {
```

Comment:
The func `Read` was not directly tested as far I could see. Becuase I've touched it (swapping order between `validate` and `setDefaults`), I've added some testing to it.